clients/js/test/updateV2.test.ts

```diff
@@ -1046,7 +1046,7 @@ test('it can change an asset collection using same update authority (delegate ex
   });
 });
 
-test('it cannot remove an asset from collection using update delegate', async (t) => {
+test('it can remove an asset from collection using update delegate', async (t) => {
```

Comment:
If my other comment is true then I think we should keep tests for not allowing asset update delegates and asset additional update delegates.